floodlight/io/skillcorner.py

```diff
@@ -0,0 +1,281 @@
+import json
+
+from typing import Dict, Tuple, Union, List
+
+import numpy as np
+import pandas as pd
+
+from floodlight import Teamsheet, Code, XY, Pitch
+
+
+def _read_json(file_path: str) -> Dict:
+    """
+    Read a JSON file from the given file path and return its content.
+
+    Parameters
+    ----------
+    file_path : str
+        The path to the JSON file to be read.
+
+    Returns
+    -------
+    dict
+        The content of the JSON file as a dictionary.
+    """
+    with open(file_path) as f:
+        return json.load(f)
```

Comment:
is that really necessary? if yes, could this become a general helper function in the io/util.py ? should add though that it's being read into memory, maybe even by calling the function ``read_json_to_memory`` and in the docstring.

Reply:
I guess there are many helper functions that could be generalized in the utils. E.g., thinking about the Kinexon parser, the `readline()` functions could very well be used in other features in the future. I would remove the function for now and put adding to the util.py on the agenda.